.pylintrc

```diff
@@ -63,6 +63,9 @@ indent-string='    '
 
 allowed-comment-prefixes=int,str,float,bool,v
 
+disallowed-functions-and-replacements-str=now=>datetime.datetime.utcnow,assertEquals=>self.assertEqual,StringIO=>python_utils.string_io,urlsplit=>python_utils.url_split,urlparse=>python_utils.url_parse,urlunsplit=>python_utils.url_unsplit,parse_qs=>python_utils.parse_query_string,unquote=>python_utils.urllib_unquote,urljoin=>python_utils.url_join,next=>python_utils.NEXT,range=>python_utils.RANGE,round=>python_utils.ROUND,str=>python_utils.convert_to_bytes or python_utils.UNICODE,zip=>python_utils.ZIP,basestring=>python_utils.BASESTRING,iteritems=>items,itervalues=>values,iterkeys=>keys,print=>python_utils.PRINT,open=>python_utils.open_file,input=>python_utils.INPUT,map=>python_utils.MAP
```

Comment:
It might be better to add a newline after each `,` ti improve the readability. The newlines then need to be properly cleaned the loaded.

Reply:
Done! The config was being trimmed before so I believe that should take care of the cleaning of the newlines.